Conceptpower+Spring/src/main/java/edu/asu/conceptpower/servlet/rest/ConceptIDLookup.java

```diff
@@ -54,10 +59,10 @@
 	 *            Holds the HTTP request information
 	 * @return XML containing concept information
 	 */
-	@RequestMapping(value = "rest/Concept", method = RequestMethod.GET, produces = MediaType.APPLICATION_XML_VALUE + "; charset=utf-8")
-    public @ResponseBody ResponseEntity<String> getConceptById(HttpServletRequest req) {
+    @RequestMapping(value = "/rest/Concept", method = RequestMethod.GET, produces = MediaType.APPLICATION_XML_VALUE
+            + "; charset=utf-8")
+    public @ResponseBody ResponseEntity<String> getConceptById(@PathVariable String id) {
```

Comment:
Pretty sure that's the wrong annotation. You want a query parameter not a part of the path.